web/server.js

```diff
@@ -123,6 +125,14 @@ function handleError(err) {
 app.get('/health', (req, res) => res.json({status: 'ok'}));
 app.get('/clientVersion', (req, res) => res.send(BUNDLE_HASH));
 
+// Translation status API
+app.get('/translationStatus/badges.svg', (req, res) => {
```

Comment:
Sure, I just did it like this because of the `/clientVersion` above - can I change it to `/client-version` while I'm at it?

Reply:
Good catch. `/clientVersion` is used by the frontend to figure out if a new version of the client is available. Changing it would mean update the frontend and would also mean causing errors for older versions of the frontend. I suggest keeping `clientVersion` as is but changing `/translationStatus` to `/transationstatus`.

What do you think?

Reply:
Sure, that makes sense! :+1: